Modix.Data/Migrations/20180716211959_Initial.cs

```diff
@@ -8,6 +9,25 @@ public partial class Initial : Migration
     {
         protected override void Up(MigrationBuilder migrationBuilder)
         {
+            migrationBuilder.CreateTable(
+                name: "BehaviourConfigurations",
+                columns: table => new
+                {
+                    Id = table.Column<int>(nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.SerialColumn),
+                    Category = table.Column<string>(nullable: false),
+                    Key = table.Column<string>(nullable: false),
```

Comment:
Would it make sense to put a unique constraint on `Category` and `Key`?

Reply:
Yes, yes it would